src/components/ImportExport.component.js

```diff
@@ -119,15 +142,20 @@ class ImportExport extends React.Component {
                 >
                     <Menu>
                         <MenuItem
-                            leftIcon={<ExportIcon />}
+                            leftIcon={<ImportIcon />}
                             primaryText={t("export")}
                             onClick={exportToCsvAndSave}
                         />
                         <MenuItem
-                            leftIcon={<ImportIcon />}
+                            leftIcon={<ExportIcon />}
                             primaryText={t("import")}
                             onClick={importFromCsv}
                         />
+                        <MenuItem
```

Comment:
It's a bit weird to have this order (Export/Import/Export empty), but I am not aware of the requirements, so I'll leave @adrianq to decide.

Reply:
Yes, I agree I have just changed the order to Import, Export and Export Empty